pkg/acs/calacs/lib/mkspt.c

```diff
@@ -47,7 +48,7 @@ int mkNewSpt (char *in_list, char *mtype, char *output) {
     ShortHdrData  stmp;
     int         nimgs, i;
     int         nx;
-	int 		extnum,nextn;
+	int 		extnum, nextn;
```

Comment:
You touch it you own it ;) Could these be initialized here?

Reply:
Both of these variables are initialized to zero in the source before they are used.  In particular, nextn is the count of the number of extensions in any single _spt.fits file so it is re-initialized for each new input file.  I slightly prefer to have my variables initialized in the place where I am about to use them, but if you think most folks like to see this done when they are declared, I can change this.